src/commands/setup.js

```diff
@@ -44,7 +45,28 @@ async function getUserInfoFromApi() {
       };
       return acc;
     }, {});
-  certSpinner.succeed(`Your certifications:\n${JSON.stringify(config.certs, null, 2)}`);
+  certSpinner.succeed('Your certifications:');
+
+  // Create a new table for certifications
+  const certsDetails = new Table({
+    head: [
+      {hAlign: 'center', content: 'id'},
+      {hAlign: 'left', content: 'project name'},
+      {hAlign: 'center', content: 'price'}],
+    colWidths: [5, 40, 7],
+  });
+
+  certifications.body
```

Comment:
You can use `config.certs` here instead, since it's already reduced. Then it would be something like:

```javascript
Object.keys(config.certs) // The keys are the project ids
  .sort((a, b) => a - b) // Sorted on keys ascending
  .forEach((id) => {
    const {name, price} = config.certs[id];
    // ... Table
});
```